build.sbt

```diff
@@ -117,7 +117,9 @@ lazy val scala = Project(id = "content-api-models-scala", base = file("scala"))
     libraryDependencies ++= Seq(
       "org.apache.thrift" % "libthrift" % "0.9.1",
       "com.twitter" %% "scrooge-core" % "4.5.0"
-    )
+    ),
+    dependencyOverrides += "org.apache.thrift" % "libthrift" % "0.9.1",
```

Comment:
adding a comment to explain why to not update (or remove the override) will help ourself in the future to not break things!